### DISCUSSION TITLE
页边距调整请教

### DISCUSSION THREAD

**Discussion by PhotometricStereo** [sjtug/SJTUThesis#835] [灌水区]
请问如何修改英文标题页面的页边距。

在Affiliation这一栏，学院名字冒出来了三个字母到下一行类似这样：

**Affiliation:         school of xxxx xxxxx-
xxx**

同样的文字在word模板里是够的：
![图片](https://user-images.githubusercontent.com/101957989/187116842-74b367b2-363b-4ff3-95e8-79e2b8023112.png)

在latex模板里有点局促了：
![图片](https://user-images.githubusercontent.com/101957989/187116621-a958b88c-bfa1-404f-9eb7-b0d88ca37c28.png)

有没有可以调整的方法呀！
非常感谢！

**Comment by PhotometricStereo** [comment]
找到了一个可行的方法，修改配置文件:
https://github.com/sjtug/SJTUThesis/blob/7868733fef46f571fc88de2cf411834842a43360/texmf/tex/latex/sjtuthesis/sjtuthesis-graduate.ltx#L146
在其中增加了一个局域环境控制，左右各扩大1mm：
```tex
\newgeometry{left=3.2cm,right=2.7cm}     %To enlarger the en title page
\begin{center}
......
\end{center}
\restoregeometry 
```

效果如下：
![图片](https://user-images.githubusercontent.com/101957989/187120646-20f1ee14-e76e-4040-988f-b615e3fef314.png)

**Reply by LogCreative** [reply]
或者直接更改 0.45 到一个更大的数值，`\textwidth` 是页面的文本宽度值。
https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/texmf/tex/latex/sjtuthesis/sjtuthesis-graduate.ltx#L159
